uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
+  </header>
+  <div class="table-responsive">
+    <table class="table">
+      <tr>
```

Comment:
The table should have colored background, check the style guide. As @AkA84 pointed out, the styleguide has priority on how things should look. So you can go back to TH